src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1264,17 +1240,66 @@ void runtime::cpu::CPU_ExternalFunction::build()
     }
 }
 
+void runtime::cpu::CPU_ExternalFunction::register_and_run_passes()
+{
+    ngraph::pass::Manager pass_manager;
+
+    pass_manager.register_pass<ngraph::pass::NopElimination>();
+    // TODO (pruthvi): Enable all the disabeled RNN fusion graph pass after fixing
+    // failing mxnet unit tests.
+    // pass_manager.register_pass<runtime::cpu::pass::LSTMFusion>();
+    // pass_manager.register_pass<runtime::cpu::pass::RNNFusion>();
+    pass_manager.register_pass<ngraph::pass::AlgebraicSimplification>();
+    // pass_manager.register_pass<runtime::cpu::pass::MultiLayerRNNFusion>();
+    // pass_manager.register_pass<runtime::cpu::pass::ConcatInputs>();
+    pass_manager.register_pass<runtime::cpu::pass::CPUBatchFusion>();
+    pass_manager.register_pass<ngraph::pass::CommonSubexpressionElimination>();
+    pass_manager.register_pass<ngraph::pass::CoreFusion>();
+    pass_manager.register_pass<runtime::cpu::pass::CPUFusion>();
+    pass_manager.run_passes(m_function);
```

Comment:
This is incorrect for dex.
If it helps, I'd suggest auditing against a clean branch to make sure more stuff like this hasn't slipped in